src/components/reusables/Page/PageHeader.module.css

```
@@ -0,0 +1,44 @@
+.page-header {
+  height: var(--page-header-height);
+  background-color: var(--component-page_header-background-color);
+  display: flex;
+  width: 100%;
+  color: var(--component-page_header-color);
+  gap: 2rem;
```

Maybe scale this down a bit? As it is, the distance between the icon and the header text is pretty large, which is particularly noticeable on small screens.

Even just downscaling to 1.5 rem helps

```suggestion
  gap: 1.5rem;
```